eng/build.yml

```diff
@@ -106,38 +98,7 @@ phases:
                   $(TestParameter)
                   --pack --publish --ci
                   --noprettyprint
-                  --configuration $(BuildConfig)
-                  $(_PublishArgs)
-                  $(_SignArgs)
-                  $(_OfficialBuildIdArgs)
-                  $(AdditionalBuildParameters)
+                  --configuration $(_BuildConfig)
+                  $(_BuildArgs)
+                  $(_AdditionalBuildParameters)
         displayName: Build
-
-    - task: PublishTestResults@1
```

Comment:
Is all this being taken care of by arcade now?

Reply:
yes